src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -33,7 +33,7 @@
 
 namespace kafka {
 
-static constexpr std::array<std::string_view, 12> supported_configs{
```

Comment:
In the commit message:

> The change is only UX, feature implementation and internal types don't
change.

That is a good to know, but please remember to describe the changes being made.